eventprocessor/processor.go

```diff
@@ -2,32 +2,38 @@ package eventprocessor
 
 import (
 	"context"
-	"fmt"
 	eventlistener "github.com/DaoCasino/platform-action-monitor-client"
 	"github.com/rs/zerolog/log"
 	"platform-backend/blockchain"
 	"platform-backend/models"
 	"platform-backend/repositories"
 	"platform-backend/usecases"
-	"time"
 )
 
 // game events
 const (
-	gameStarted eventlistener.EventType = iota
-	actionRequest
-	signidicePartOneRequest
-	signidicePartTwoRequest
-	gameFinished
-	gameFailed
-	gameMessage
+	gameStarted             = 0
```

Comment:
iota ?

Reply:
I don't like iota in cases like that. Original events numeration controlled by game contract. Contract doesn't  guaranties that order is without gaps and also doesn't guaranties that new event will have "next" number as  type.

Reply:
oh, I see